source/common/router/rds_subscription.cc

```diff
@@ -20,6 +20,8 @@ RdsSubscription::RdsSubscription(
     : RestApiFetcher(cm, rds.config_source().api_config_source().cluster_names()[0], dispatcher,
```

Comment:
Does everywhere that has a RestApiFetcher have an api_config_source at this point? Can we just pass the API config source to the fetcher? I think that would make a bunch of callsites simpler and push the other logic inside the fetcher?

Reply:
Addressed in 1627b53.